gcp/scripts/run_pg_refresh_pg_data_local_db.sh

```diff
@@ -0,0 +1 @@
+DB_HOST=localhost DB_PORT=15432 DB_PASSWORD=password INSTANCE_CONNECTION_NAME= python3.11 ../cloud-functions/refresh-pga-data/main.py --write --data-types schedule leaderboard players
```

Comment:
some helper scripts for running things with a local DB (start_db.sh below starts the DB)